community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/CompatibilityCache.scala

```diff
@@ -67,20 +70,31 @@ class CommunityCompatibilityFactory(graph: GraphDatabaseQueryService, kernelMoni
       v3_1.CostCompatibility(graph, as3_1(config), CompilerEngineDelegator.CLOCK, kernelMonitors, log, spec.planner, spec.runtime, spec.updateStrategy)
   }
 
+  override def create(spec: PlannerSpec_v3_3, config: CypherCompilerConfiguration) =
+    (spec.planner, spec.runtime) match {
+      case (CypherPlanner.rule, _) =>
+        throw new InvalidArgumentException("The rule planner is no longer a valid planner option in Neo4j 3.3. If you need to use it, please compatibility mode Cypher 3.1")
```

Comment:
Typo in the message "If you need to use it, please `select` compatibility mode Cypher 3.1"